code/controllers/subsystem/air.dm

```diff
@@ -44,6 +45,12 @@ SUBSYSTEM_DEF(air)
 	var/map_loading = TRUE
 	var/list/queued_for_activation
 
+	// VORELINDA config options (am I really calling it that?)
+	// The below two refer to how many SSair_turf ticks will go by before a turf will run again.
```

Comment:
I'm getting prob(80) vibes

Reply:
Thing is `prob(80)` has no fixed, consistent (per tick) frequency, unlike here.

Reply:
This is there primarily to artificially slow down the simulation. Without it comes quite a few gameplay issues that come from TG's map design... *really* not being meant for ZAS propagation speeds